sdk/src/DocuSign.eSign/DocuSign.eSign.nuspec

```diff
@@ -1,41 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
@RajRele is this intended? How will Nuget release work?

Reply:
https://github.com/docusign/docusign-esign-csharp-client/pull/338#discussion_r674993203